lib/hpc/utils.pm

```diff
@@ -93,13 +93,17 @@ sub setup_scientific_module {
     my $mpi = get_required_var('MPI');
 
     if (get_var('HPC_LIB') eq 'scipy') {
-        zypper_call("in python3-scipy-gnu-hpc");
-        assert_script_run("env MPICC=mpicc python3 -m pip install mpi4py");
+        # $mpi-gnu-hpc-devel will be installed on compute nodes
+        # which is required to compile the mpi4py.
+        # This seems is not be shared by NFS
```

Comment:
I don't understand what this statement means.

Reply:
Controller and compute nodes should have the *-gnu-hpc packages installed. As per design it s supposed the controller shares `/usr/lib/hpc` via NFS, in order to avoid re-installing HPC packages on compute nodes. But compiling mpi4py, i found that i needed $mpi-gnu-hpc-devel. i would expect that those packages are found in `/usr/lib/hpc` but likely this is not the case.

Reply:
so don't we test the NFS sharing scenario for HPC clusters?

Reply:
NFS will be covered in https://progress.opensuse.org/issues/126698

Reply:
okay, Thanks @czerw !